src/felles-komponenter/utils/ekstern-lenke-ikon.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import React from 'react';
+
+export default function EksternLenkeIkon() {
```

Comment:
Er ikke dette litt rart? Bedre å bare bruke svg-en direkte? Bruken blir vel den samme, bare at du slipper å wrappe den i en react component ?

Reply:
For games react versjon, funker bare med react 16